crates/lib3h/src/gateway/gateway_transport.rs

```diff
@@ -21,6 +21,9 @@ impl<'gateway> Transport for P2pGateway<'gateway> {
     // TODO #176 - Return a higher-level uri instead?
     fn connect(&mut self, uri: &Url) -> TransportResult<ConnectionId> {
         trace!("({}).connect() {}", self.identifier, uri);
+        if &self.identifier != "__network__" {
```

Comment:
had to look up this magic string, could you import `NETWORK_GATEWAY_ID` instead?

Reply:
aw, crap... that's debugging that shouldn't be in there... we don't want to `std::process::exit(127)` ; )

Reply:
removed